padertorch/contrib/examples/source_separation/pit/templates.py

```diff
@@ -29,17 +29,7 @@
 \t\t--stderr=%x.%reqid.err \\
 \t\t--tracefile=%x.%reqid.trace \\
 \t\t-N evaluate_{nickname} \\
```

Comment:
Can you rename `nickname` to `experiment_name` to match the naming in the Python script?